### PR TITLE
Translation 3.4

### PR DESCRIPTION
**Pull request by lucernae**
CC @ismailsunni 
Some question for this:
- What happened when we 'tr' a dynamic text? Does the base string is generated?
- Alternative way is to extract the specially formatted strings that needs to be translated manually. (Put the generated text manually to transifex?)

### CODE REVIEW

**Comment by ismailsunni** [comment]
@lucernae 
- AFAIK, the dynamic translation is not working for now.
- Manually adding text to transifex is prone to missing text, and I think it will be hard to manage

File: safe/gui/tools/help/osm_downloader_help.py [hunk 1]
```diff
@@ -98,17 +98,19 @@ def content():
...
             'options panel will be enabled. The panel lets you select which '
             'admin level you wish to download. The admin levels are country '
             'specific. When you select an admin level, the local name for '
             'that admin level will be shown. You can change which country '
             'is used for the admin level description using the country drop '
             'down menu. The country will be automatically set to coincide '
-            'with the view extent if a matching country can be found.')))
+            'with the view extent if a matching country can be found.') %
+        (
+            m.ImportantText(tr('Political boundaries')).to_html(),
```
Comment (timlinux) [inline safe/gui/tools/help/osm_downloader_help.py]: But if we do this we are breaking separation between content and markup - now we will html embedded in the string....I think that is bad...

**Comment by timlinux** [comment]
Ok I have some reservations about this because this approach embeds html markup into the strings which is against the design intent of the messaging framework. But I can't see a more elegant way to deal with this right now so I suggest to go ahead and merge here and then we can figure out a more elegant approach in the future.

**Comment by ismailsunni** [comment]
Here the example of the problem that we found:
```
    message.add(m.Paragraph(tr(
        'There are three main areas to the dock panel:')))
    bullets = m.BulletedList()
    bullets.add(m.Text(
        tr('the '),
        m.ImportantText(tr(
            'questions ')),
        tr('area')))
    bullets.add(m.Text(
        tr('the '),
        m.ImportantText(tr(
            'results ')),
        tr('area')))
    bullets.add(m.Text(
        tr('the '),
        m.ImportantText(tr(
            'buttons ')),
        tr('area')))
```
Then in transifex we got :
screen shot 2016-05-24 at 21 46 05
It's very difficult to translate word by word in Indonesia or French. In Indonesia, we don't have `the`, and we have different order of word.

**Pull request merged**